documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -610,6 +610,12 @@ NOTE: This list is not exhaustive - there may be codes in use that are not in th
 | 512MB
 | Sony UK
 
+| b04170
+| 5
+| 1.0
```

Comment:
1.1, see above.

Reply:
```suggestion
| 1.1
```

Reply:
Looks like Tim confirmed this is an R0 1.0 board, so I'll switch this back. Read the comments in the wrong order.